### ISSUE TITLE
readonly should be able lookup item with large number of options

### ISSUE THREAD

**Issue by sunnysideup** [silverstripe/silverstripe-framework#10320]
We have over 50,000 members so we can not create a dropdown lookup, fair enough, but when a readonly transformation take place maybe it could look it up anyway. 

![image](https://user-images.githubusercontent.com/167154/168447145-f3b248b4-3914-4518-87d8-8230961dacac.png)

**Comment by michalkleiner** [comment]
Fair enough, that's a good point. Been in a similar situation where we have a read-only field but a large underlying collection and the IDs provide nearly no value unless you know how to get to the actual value.
